Composer/packages/extensions/visual-designer/src/components/menus/IconMenu.tsx

```diff
@@ -96,6 +96,7 @@ export const IconMenu: React.FC<IconMenuProps> = ({
     <OverflowSet
       // @ts-ignore
       styles={{ position: 'absolute', top: 0 }}
+      aria-lable="icon menu"
```

Comment:
`aria-label` - spelling
I recommend `aria-label="menu"

The screen reader will say, 'Menu button'

Reply:
Sure,